src/global_styling/mixins/_form.scss

```diff
@@ -262,16 +262,21 @@
 
     // Needs to be set for autofill
     // sass-lint:disable-block no-vendor-prefixes
-    &:-webkit-autofill {
-      -webkit-text-fill-color: lightOrDarkTheme($ouiColorDarkestShade, $ouiColorLightShade);
+    &:-webkit-autofill,
+    &:autofill {
+      /* Many browsers use `!important` in their built-in `:-webkit-autofill` style declarations,
+       * making their colors non-overridable. `-webkit-text-fill-color` is able to overwrite the
+       * appearance of texts and is used here as a workaround.
+       */
+      -webkit-text-fill-color: $ouiColorDarkestShade;
```

Comment:
Why are we using the shade definition directly, and not `$ouiTextColor`?

Reply:
`ouiColorDarkestShade` is what the current stylesheet uses and the value of `ouiTextColor` doesn't match `ouiColorDarkestShade` in dark themes.

Reply:
I intended this comment for the other file 😁